### DISCUSSION TITLE
Newbie, terminal output, no voltage

### DISCUSSION THREAD

**Discussion by nullbreached** [David00/rpi-power-monitor#148] [General]
I’m dealing with some issues with setup, the data in the terminal out is all <1 for all parameters but more confusingly, the voltage for CT1 is not displaying anything at all, it’s blank.  CT2 is displaying data (albeit erroneous).

**Comment by David00** [comment]
Hi @nullbreached, there's a couple of things to check for this -

1. Make sure that the power monitor is not running in the background when using terminal mode:

```bash
# Check with: 
sudo systemctl status power-monitor.service

# If it says "active (running)", stop it with: 
sudo systemctl stop power-monitor.service
```

2. Generate a plot to show the raw data coming in.  See [--mode plot](https://david00.github.io/rpi-power-monitor/docs/v0.3.0/advanced-usage.html#--mode) in the docs.


Some additional questions I have to help guide the next troubleshooting direction are:
* What model Pi are you using?
* Did you flash my custom OS image, or do the manual installation? If manual install, which distro did you use?
* Which power monitor PCB are you using? Or, is this something else like a breadboard setup or self-fabrication?

**Comment by nullbreached** [comment]
So I’ll confess, I totally overlooked the fact that the power adapter was AC! LOL it explains a lot. I’ve made progress but I’m uncertain how to adjust the calibration when the values in the terminal fluctuate, example, 18.3-18.8. How do I determine the value to calibrate against?On Nov 30, 2024, at 8:39 PM, David Albrecht ***@***.***> wrote:﻿
Hi @nullbreached, there's a couple of things to check for this -

Make sure that the power monitor is not running in the background when using terminal mode:

# Check with: 
sudo systemctl status power-monitor.service

# If it says "active (running)", stop it with: 
sudo systemctl stop power-monitor.service

Generate a plot to show the raw data coming in.  See --mode plot in the docs.

Some additional questions I have to help guide the next troubleshooting direction are:

What model Pi are you using?
Did you flash my custom OS image, or do the manual installation? If manual install, which distro did you use?
Which power monitor PCB are you using? Or, is this something else like a breadboard setup or self-fabrication?

**Reply by David00** [reply]
Haha, it's all good!

The fast update rate in the terminal can definitely make it tricky to calibrate. One strategy is to take the average of the values you're seeing - so 18.55 - and use that as the power monitor's reading when comparing against your handheld meter.

A slightly easier way is to use the Individual CT Dashboard in Grafana (should already be preloaded in your Grafana assuming you used the custom OS image) and look at the amperage readings for the CT you're using.  Set the time interval in Grafana to "Last 3 Hours" and it will use about a 10 second average value for each individual sensor.  And, Grafana will auto refresh as fast as every 5 seconds by default, which really helps slow down the amount of data coming your way when calibrating.

![image](https://github.com/user-attachments/assets/0803e86e-2cbf-43e6-a4f0-845bca2456f7)

**Comment by nullbreached** [comment]
I just placed an order for a couple more SCT. I bought the 60A of the 50Amp hot tub line but I’m probably still have that wrong in my head and it should just be a 30A (Does a dual-line 50A line have a max of approximately 25A per line and thus I don’t need a 50A or 60A SCT? (If so, substitute in my order. The 100A is for my main.)

**Reply by David00** [reply]
> (Does a dual-line 50A line have a max of approximately 25A per line and thus I don’t need a 50A or 60A SCT? 

No, each leg can pull up to 50A. You are good to go with the 60A sensor (or the 50A sensor that I just added to the shop this morning!).  If you want to switch to the 50A, I can update your order and do that instead - just let me know :)

**Comment by nullbreached** [comment]
If it’s not a pain and would make more sense, sure. Switch in the 5A. 

But now that you’ve mentioned it how does that impact the 100A SCT I’m using for my main … I think the panel is 200A?
